feat/final-words (#28)
computer-aided-biotech/better-cb: content/07.conclusion.md

## Final words

> The importance of good practices in computational biology research have gain recognition among researchers given that new advice has been published in recent years and cloud-based Git repositories have become the default platform to share code and data analyses. This work adds to the existing literature, introducing a comprehensive list of best practices that can be applied to any computational biology project, regardless of the specific subfield and the experience of the researcher. We sought to comprehensively introduce tools covering the whole computational biology "journey", serving as a reference guide to be reached when needed. This approach complements other efforts that dive into specifics tools and that you can reach to get a better understanding of the tools you deem relevant to your research.

Reviewer: ```suggestion
Good practices in the computational biology field have gained recognition among researchers thanks to several guidelines published in recent years, and cloud-based Git repositories becoming the default platform to share code and data analyses. This review adds to the existing literature, introducing a comprehensive list of best practices that can be applied to any computational biology project, regardless of the specific subfield and the experience of the researcher. We sought to comprehensively introduce tools covering the whole computational biology "journey", serving as a reference guide to be re-visited when needed. This approach complements other efforts that provide a deeper understanding of specifics tools that you might deem relevant to your research.
```

> Although we have introduced many tools, we are aware that other good practices may have been left behind. Moreover, as time progresses, practices will expand and change as new tools emerge and the field matures. Both, the many tools and practices introduced and their changing nature, may seem very overwhelming, especially for someone new to the field. To overcome this, we encourage you to implement a few of practices first, starting from your personal research, and expand your repertoire over time. More importantly, keep a mindset of striving for reproducibility, regardless of the tools used.

Reviewer: ```suggestion
Although we have introduced many tools, we are aware that other good practices were probably left behind. Moreover, as time progresses, practices will expand and change as new tools emerge and the field matures. This means that updated versions of reviews like this one will always be essential for the novice community. More importantly, the many tools and practices introduced in this study, and their ever-changing nature, may seem overwhelming, especially for someone new to the field. To overcome this, we encourage you to implement only a few practices first, starting from your personal research, and over time expand your repertoire. More importantly, keep a mindset of striving for reproducibility, regardless of the tools used.
```

> In the midst of our work, under the many pressures of scientific research, we can forget about keeping good practices in place. In such scenario, the consequences will be not seeing immediately, but the long-term sustainability of our research will be compromised. Computational biology research heavily relies on building new knowledge from previous one. In that regard, the practices that we adopt in our personally are the building blocks of the overall reproducibility of the field that propel computational biology research forward.

Reviewer: ```suggestion
In the midst of our work, under the many pressures of scientific research, we can forget to keep good computational practices in place. In such scenario, the consequences will not be seen immediately, but the sustainability of our research will be compromised in the long run. Computational biology research heavily relies on building new knowledge from previous one; in that regard, the personal practices that we adopt will act as building blocks for the overall reproducibility of the field, and will propel computational biology research forward.
```
